Jellyfin.Plugin.Kitsu/Providers/KitsuIO/ApiClient/Models/Titles.cs

```diff
@@ -0,0 +1,30 @@
+using System.Text.Json.Serialization;
+
+namespace Jellyfin.Plugin.Kitsu.Providers.KitsuIO.ApiClient.Models
+{
+    public class KitsuTitles
+    {
+        [JsonPropertyName("en")] public string En { get; set; }
+
+        [JsonPropertyName("en_jp")] public string EnJp { get; set; }
+
+        [JsonPropertyName("ja_jp")] public string JaJp { get; set; }
+
+        [JsonPropertyName("en_us")] public string EnUs { get; set; }
+
+        public string GetTitle =>
+            !string.IsNullOrWhiteSpace(En) ? En :
+            !string.IsNullOrWhiteSpace(EnUs) ? EnUs :
+            !string.IsNullOrWhiteSpace(EnJp) ? EnJp :
+            JaJp;
+
+        public bool Equal(string title)
+        {
+            return
+                (En?.Equals(title) ?? false) ||
+                (EnUs?.Equals(title) ?? false) ||
+                (EnJp?.Equals(title) ?? false) ||
+                (JaJp?.Equals(title) ?? false);
```

Comment:
I know this is 1to1 the old statement, but could we not use a [`switch` expression](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression) here on `title` to have less null checks and possibly easier to read logic?

Not blocking and maybe that is just me finding switch stamens more elegant ¯\_(ツ)_/¯

Reply:
We can't use switch expression here since it requires the case expressions to be constant values.
But I get what you are saying, the code doesn't look great here, so I updated the title logic:
Instead of using this massive conditional expression I've added all the titles into a list and used linq to filter out the invalid titles, take a look ant tell me what you think.

Reply:
It's not blocking for me but something I would like your or another plugin maintainers view on: 
Is Linq not a tat overkill here and causes unnecessary allocations?

I wasn't a fan of the long statement, but I am a tat worried about the performance implication of using Linq here.